code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -668,10 +671,12 @@ GLOBAL_VAR(bomb_set)
 				return
 	return
 
-//==========DAT FUKKEN DISK===============
+//////////////////////////////
+// MARK: DAT FUKKEN DISK
+//////////////////////////////
 /obj/item/disk/nuclear
-	name = "nuclear authentication disk"
-	desc = "Better keep this safe."
+	name = "\improper Nuclear Authentication Disk"
```

Comment:
IMO, we should have no capitalization for objects, unless its like X-01 where the capitalization is part of the designation, as its not really a proper noun